lab_1_keywords_tfidf/main.py

```diff
@@ -2,22 +2,35 @@
 Lab 1
 Extract keywords based on frequency related metrics
 """
-from typing import Optional, Union
+from typing import Optional, Union, List
```

Comment:
instead of using List, consider upgrading to 3.9+ version of python

Reply:
Done.